src/main/PersistentState.h

```diff
@@ -46,9 +46,9 @@ class PersistentState
     setSCPStateV1ForSlot(uint64 slot, std::string const& value,
                          std::unordered_map<Hash, std::string> const& txSets);
 
-    bool shouldRebuildForType(LedgerEntryType let);
-    void clearRebuildForType(LedgerEntryType let);
-    void setRebuildForType(LedgerEntryType let);
+    bool shouldRebuildForOfferTable();
```

Comment:
`kDBBackend` can be removed now (this doesn't need to be a migration, but we need to delete that row)

Reply:
I'd like to keep it in until the next release. Reason is `maybeUpgradeToBucketListDB` gets called on the startup path and checks `kDBBackend`. If it's not BucketListDB, it drops a bunch of tables. We can't do this unconditionally on startup as some operators have separate users for DB maintenance and call `upgrade-db` from a separate user. After the migration, we can drop this.